test/build/legacy-mount-scripts/legacy-mount.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+const {setupBuildTest} = require('../../test-utils');
+
+const cwd = path.join(__dirname, 'build');
+
+describe('config: mount scripts (legacy)', () => {
+  beforeAll(() => {
+    setupBuildTest(__dirname);
+  });
+
+  it('mounted ./src', () => {
+    const js = path.join(cwd, '_dist_', 'index.js');
```

Comment:
nit: since most tests start as copy-paste, I'd actually want to prioritize a pass on all of these to clean them for nits. Example:

- `js` is a vague variable name, we use `*Loc` in the rest of the codebase to signify a file path
- test names: when tests are this focused, I generally like to see "does X when Y" naming to make it really clear what's being tested (hard to tell from a quick read for some of these)
- `cwd` isn't really the right term here, we want the test output directory, not the current working directory. Actually, could `setupBuildTest` return that variable so that we turn the implicit connection of "oh, when setupBuildTest is done, I know that this other folder on disk now exists" into something more explicit?